### DISCUSSION TITLE
GitHub star list new feature experience

### DISCUSSION THREAD

**Discussion by zhicheng-ning** [X-lab2017/open-wonderland#105] [1️⃣ 新鲜事]
一、在个人主页的 Stars 页面下，可以点击 Create list 创建一个分组
![image](https://user-images.githubusercontent.com/39022409/175817529-04be19dd-fc21-4efb-b7c3-c95c27743e07.png)

二、对以往 star  了的项目进行分类，可以将属于同一技术领域内的项目添加到上面创建好的分组中去

![image](https://user-images.githubusercontent.com/39022409/175817654-e028ff07-317c-4e4a-a780-dcf6032a5f8a.png)

三、个人思考
以一个音乐平台为例做以下类比：
仓库 —— 歌曲
star list —— 歌单

在音乐软件中，自媒体或者个人音乐号，会按照某一种分类标准，比如说将相似风格的歌曲放在一起作为一个歌单，这些歌单会推荐给喜欢这种歌曲风格的用户。而 GitHub 目前的 star list 属于 beta 功能，个人创建的 star list 仅仅自己本人可见，无法分享给其他用户。但是我们可以预见的是 GitHub 官方后续肯定会添加 star list 的分享，收藏，推荐等等功能，让我们拭目以待~

**Comment by tyn1998** [comment]
又在Discussions里涨知识了！

**Comment by xgdyp** [comment]
这个功能出了一段时间了，之前我一直就觉得是个分类的功能，看这么一说感觉很有道理！
